src/components/layout/NavMenu.tsx

```diff
@@ -195,7 +197,8 @@ const NavMenu = (props: any) => {
         alignContent="center"
         justify="center"
       >
-        <img src={appLogo} alt="logo" className={classes.logo} />
+        {/* <img src={appLogo} alt="logo" className={classes.logo} /> */}
```

Comment:
It should be safe to delete this line since we are replacing it with the line below it.